spec/controllers/concerns/hyrax/works_controller_behavior_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+RSpec.describe Hyrax::WorksControllerBehavior, type: :controller do
+  subject(:controller) { controller_class.new }
+  let(:paths)          { Rails.application.routes.url_helpers }
+  routes               { Rails.application.routes }
+
+  let(:controller_class) do
```

Comment:
Does it make sense to use a rspec-defined controller instead of fully declaring it?  Rspec takes care of the cleanup of the classes then.  Here's an example with custom routes: https://relishapp.com/rspec/rspec-rails/v/3-9/docs/controller-specs/anonymous-controller#draw-custom-routes-for-defined-controllers

Reply:
probably. I didn't know about this RSpec feature and can take a look.

Reply:
so, it's not clear to me how to define a controller with a mixin and draw routes more succinctly using this interface. it looks to me like we would still need to define a named controller(?)

can you help prod me in the right direction?